.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13-dev"]
```

Comment:
Happy to take it as we've lucked out. We should do this in dedicated PRs in future in case there is fallout to deal with.